src/test/java/be/seeseemelk/mockbukkit/ServerMockTest.java

```diff
@@ -183,6 +183,24 @@ void getBukkitVersion_NotNull()
 		assertNotNull(server.getBukkitVersion());
 	}
 
+	@Test
+	void getBukkitVersion_CorrectPattern()
+	{
+		assertTrue(server.getBukkitVersion().matches("1\\.[0-9]+(\\.[0-9]+)?-R[0-9].[0-9]-SNAPSHOT"));
```

Comment:
If you browse the [SpigotMC Nexus repository](//hub.spigotmc.org/nexus/#view-repositories;snapshots~browsestorage), you’ll see that revision numbers aren’t mandatory, such as in `1.14.3-pre4-SNAPSHOT`.

Reply:
Perhaps ignoring what's between the Minecraft version and SNAPSHOT (and maybe after?) would be better?

Reply:
I didn’t even catch the `-pre4`, so yeah, I agree.